pkg/jsonapi/jsonapi.go

```diff
@@ -134,6 +134,31 @@ func DataRelations(c echo.Context, statusCode int, refs []couchdb.DocReference,
 	return json.NewEncoder(resp).Encode(doc)
 }
 
+// DataRefences can be called to send a References page,
```

Comment:
I would prefer to reuse the `DataRelations` function by changing the `total` parameter by a `meta` parameter.

Reply:
If you think it is a too big change for this PR, at least we should fix the typo: s/DataRefences/Data/References/g

Reply:
It felt weird to have a very long signature and passing `nil` parameters in `referencedby.go` but if you prefer limiting the number of functions, I can do that as well.